gef.py

```diff
@@ -6772,11 +6772,10 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
 
         args : argparse.Namespace = kwargs["arguments"]
         if args.registers and args.registers[0]:
-            requested_regs = set(args.registers)
-            valid_regs = set(gef.arch.all_registers) & requested_regs
-            if valid_regs:
-                regs = valid_regs
-            invalid_regs = requested_regs - valid_regs
+            all_regs = gef.arch.all_registers
```

Comment:
we shouldn't need to revert this part at all. It's only used to check for errors

Reply:
Never mind.